src/start_fact_backend.py

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env python3
-'''
+"""
```

Comment:
I'd be really happy if we could refrain from doing unrelated ruff fixes in commits.
All those fixes will eventually (when #987 is merged) land in a single commit in #1045

Reply:
merged master after the merge of #1045